### PR TITLE
Content and layout changes

### PR DESCRIPTION
**Pull request by lindsayallan**
Adding all content changes in

### CODE REVIEW

File: app/views/course-2024/index.njk [hunk 1]
```diff
@@ -118,115 +197,214 @@
...
+          <ul class="govuk-list govuk-list--bullet">
+            <li>a bachelor’s degree class 2:2 (honours) or higher</li>
+            <li>a master’s degree</li>
+            <li>a PhD</li>
+          </ul>
         </p>
+
+        <p class="govuk-body">If you are eligible you will receive the bursary automatically, you do not need to apply for it.</p>
+
+        <p class="govuk-body"><a href="">Find out whether you are eligible for a bursary</a></p>
```
Comment (lsalivonaite-transform) [inline app/views/course-2024/index.njk:350]: Link to git?

File: app/views/course-2024/index.njk [hunk 2]
```diff
@@ -286,41 +464,34 @@
...
+  <h2 class="govuk-heading-l heading--border-bottom">Give your application the best chance of success</h2>
+ <div class="govuk-notification-banner govuk-notification-banner--git" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner" style="border-left-color: #f9b400;">
     <div class="govuk-notification-banner__content govuk-notification-banner__content--grey">
-      <p class="govuk-heading-m">Support and advice</p>
+      <h3 class="govuk-heading-s">Get support and advice from a teacher training adviser</h3>
+
       <p class="govuk-body">
-        You can <a href="https://getintoteaching.education.gov.uk/teacher-training-advisers" class="govuk-link govuk-link--no-visited-state">get a teacher training adviser</a> or <a href="https://getintoteaching.education.gov.uk/help-and-support">contact Get Into Teaching</a> to get free support.
+        <a href="#">Get free, ongoing one-to-one support from an adviser</a> with years of teaching experience. They can help you find a course, explain funding options and guide you step-by-step through the application process.</p>
```
Comment (lsalivonaite-transform) [inline app/views/course-2024/index.njk:479]: Link to GIT - TTA?

File: app/views/course-2024/index.njk [hunk 3]
```diff
@@ -118,115 +197,214 @@
...
+            <li>a master’s degree</li>
+            <li>a PhD</li>
+          </ul>
         </p>
+
+        <p class="govuk-body">If you are eligible you will receive the bursary automatically, you do not need to apply for it.</p>
+
+        <p class="govuk-body"><a href="">Find out whether you are eligible for a bursary</a></p>
+
+        <p class="govuk-body">If you're a non-UK citizen without indefinite leave to remain, you will usually not be eligible for a bursary. <a href="">Find out about fees and funding for non-UK citizens.</a></p>
```
Comment (lsalivonaite-transform) [inline app/views/course-2024/index.njk:352]: link to GIT?

**Review by lsalivonaite-transform — APPROVED** [review]
Excellent, just add a couple of GIT links on some of the empty hrefs!

**Review by RebeccaCryan — APPROVED** [review]

**Pull request merged**